data/json/items/appliances.json

```diff
@@ -153,6 +153,33 @@
     "volume": "260 L",
     "weight": "300 kg"
   },
+  {
+    "id": "autoclave",
+    "type": "TOOL",
+    "name": { "str": "autoclave" },
+    "description": "This is an autoclave.  It's useful for sterilizing things like CBMs.  It requires a massive amount of power, so will need to be plugged into a power grid.",
```

Comment:
```suggestion
    "description": "This is an autoclave.  It's useful for sterilizing things like CBMs.  It requires a massive amount of power, so it will need to be plugged into a power grid.",
```